plugins/woocommerce-admin/client/customize-store/transitional/index.tsx

```diff
@@ -122,15 +122,13 @@ export const Transitional = ( {
 					) }
 
 					<Button
+						href={ homeUrl }
 						className="woocommerce-customize-store__transitional-preview-button"
-						variant={
-							isEntrepreneurFlow() ? 'secondary' : 'primary'
-						}
+						variant="link"
```

Comment:
Let's keep the look and feel the same as it was. I only meant switching from `window.open( ... )` to `href`. Apologies I didn't explain the intention fully.

Reply:
@kmanijak Thanks for the feedback, I have implemented the same.